app/src/androidTest/java/co/nimblehq/compose/crypto/ui/BaseScreenTest.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.compose.crypto.ui
+
+import co.nimblehq.compose.crypto.test.CoroutineTestRule
+import co.nimblehq.compose.crypto.test.runBlockingTest
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineScope
+import org.junit.Rule
+
+@ExperimentalCoroutinesApi
+abstract class BaseScreenTest {
+
+    @get:Rule
+    private var coroutineRule = CoroutineTestRule()
+
+    protected fun runBlockingTest(block: suspend TestCoroutineScope.() -> Unit) =
+        coroutineRule.runBlockingTest(block)
```

Comment:
After migrating, I don't think we need to keep using `runBlockingTest`, use `runTest` instead.